Cargo.toml

```diff
@@ -40,17 +41,20 @@ hot_reload = []
 buffer_labels = []
 
 [dependencies]
+bytemuck = { workspace = true }
+fello = { workspace = true }
+peniko = { workspace = true }
 wgpu = { workspace = true }
 raw-window-handle = "0.5"
 futures-intrusive = "0.5.0"
 parking_lot = "0.12"
-bytemuck = { version = "1.12.1", features = ["derive"] }
 smallvec = "1.8.0"
-fello = { git = "https://github.com/dfrg/fount", rev = "a8c0686ca9da236420c04d1f476c41cf7fe6d2ba" }
-peniko = { git = "https://github.com/linebender/peniko", rev = "cafdac9a211a0fb2fec5656bd663d1ac770bcc81" }
-guillotiere = "0.6.2"
+vello_encoding = { path = "crates/encoding" }
 
 [workspace.dependencies]
+bytemuck = { version = "1.12.1", features = ["derive"] }
+fello = { git = "https://github.com/dfrg/fount", rev = "a8c0686ca9da236420c04d1f476c41cf7fe6d2ba" }
```

Comment:
This will need to be updated to the match the current rev in main to resolve the merge conflict.